src/uk/gov/hmcts/contino/TeamNames.groovy

```diff
@@ -60,4 +61,19 @@ class TeamNames {
       .replace(" ", "-")
   }
 
+  def getDefaultSlackChannel(String product) {
+    def teamNames = getTeamNamesMap()
+    if (!teamNames.containsKey(product) || !teamNames.get(product).get(DEFAULT_SLACK_CHANNEL_KEY) || teamNames.get(product).get(DEFAULT_SLACK_CHANNEL_KEY).isEmpty()) {
+      throw new RuntimeException(
+        "defaultSlackChannel is not configured for Product ${product}"
+          + "Please create a PR to update team-config.yml in cnp-jenkins-config."
+      )
+    }
+    return teamNames.get(product).get(DEFAULT_SLACK_CHANNEL_KEY)
+  }
+
+  def getSlackChannel(String product, String slackChannel) {
```

Comment:
kind of seems like TeamNames should have the pipeline config in its constructor so that you don't need to pass this slack channel here, WDYT?